piqe_ocp_lib/api/resources/ocp_deploymentconfigs.py

```diff
@@ -166,6 +144,9 @@ def list_all_deployments_in_a_namespace(self, namespace):
         """
         api_response = None
         try:
+            # if is_deployment:
+            #     api_response = self.ocp_deployment.get(namespace=namespace)
+            # else:
             api_response = self.ocp_dcs.get(namespace=namespace)
         except ApiException as e:
```

Comment:
Moving forward we should raise the exception so the calling method can make a determination on whether to act or not.

Reply:
Sure